tests/net/err.rs

```diff
@@ -93,9 +98,14 @@ where
             CrankError::TimeLimitHit(lim) => {
                 write!(f, "Time limit of {} seconds exceeded.", lim.as_secs())
             }
-            CrankError::Fault(fault) => {
-                write!(f, "Node {:?} is faulty: {:?}.", fault.node_id, fault.kind)
-            }
+            CrankError::Fault {
+                reported_by_id,
+                fault,
+            } => write!(
+                f,
+                "Node {:?} reported another node {:?} as faulty: {:?}.",
```

Comment:
Is "another" always true by the way? Technically, a node could report itself as being faulty, which would be strange, but all the more reason to fail a test!

Reply:
Maybe even mention that the reporting node was correct!